April 12 Office Add-ins Community Call Blog Post (#1271)
pnp/blog: content/office-add-ins-community-call/2023-04-12/index.md

---
title: "Office Add-ins developer platform community call – April 12th, 2023"

> summary: "This month's community call topics include:  Word Extensibility Update – April 2023 presented by Yun Wang, Principal Product Manager at Microsoft and Top 5 AppSource Validation Errors (December 2022-February 2023) presented by Elizabeth Samuel, Senior Technical Writer at Microsoft. Five questions are formally addressed in this month’s Q&A, other questions addressed in chat throughout call. The call was hosted by Preethika Kiruveedula, Product Manager at Microsoft. Recorded on April 12, 2023.."

Reviewer: ```suggestion
summary: "Topics include:  Word Extensibility Update presented by Yun Wang, Principal Product Manager at Microsoft and Top 5 AppSource Validation Errors presented by Elizabeth Samuel, Senior Technical Writer at Microsoft. Call hosted by Preethika Kiruveedula, Product Manager at Microsoft. Recorded on April 12, 2023."
```
This metadata is for SEO so best to remove unnecessary stuff!

date: 2023-04-12T08:01:00-05:00
author: "Preethika Kiruveedula"
githubname: preethikakiru
categories: ["Office add in developer community call"]
images:
- images/Session-Promo-Apr-2023.jpg
tags: ["Office Add-ins","Microsoft 365"]
type: "regular"
videos:
- https://youtu.be/yvJ8n6GaO2s
draft: true
---

# Office Add-ins developer platform community call - April 12, 2023

## This month's agenda and presenters

The call was hosted by [Preethika Kiruveedula](www.linkedin.com/in/preethika-kiruveedula-529b7a148), Product Manager, Microsoft.

* **Word Extensibility APIs (Preview) - April 2023.** [Yun Wang](https://www.linkedin.com/in/wang-yun-99370463/?originalSubdomain=cn), Principal Product Manager, Microsoft.
* **Top 5 AppSource Validation Errors December 2022 - February 2023.**[Elizabeth Samuel](https://www.linkedin.com/in/elizabeth-samuel-9a1629153/), Senior Technical Writer, Microsoft. 
* **Q&A.** See question and answers at end of call and in chat throughout call. [Preethika Kiruveedula](www.linkedin.com/in/preethika-kiruveedula-529b7a14), Product Manager, Microsoft.

{{< youtube yvJ8n6GaO2s >}}

## View video segments

* Introduction [00:00](https://youtu.be/yvJ8n6GaO2s?t=0)

> * Word Extensibility Update for April 2023  [00:46](https://youtu.be/yvJ8n6GaO2s?t=46)

Reviewer: ```suggestion
* Word Extensibility Update for April 2023 [00:46](https://youtu.be/yvJ8n6GaO2s?t=46)
```

* Top 5 AppSource Validation Errors during December 2022 to February 2023 [11:13](https://youtu.be/yvJ8n6GaO2s?t=673)
* Q&A [20:00](https://youtu.be/yvJ8n6GaO2s?t=1200)
* Resources [24:37](https://youtu.be/yvJ8n6GaO2s?t=1477)
* Closing [25:33](https://youtu.be/yvJ8n6GaO2s?t=1533)

## Topic summaries

> * **Word Extensibility Updates - April 2023.** Results and insights from 2 surveys. Structured document content usage in Word add-ins. Questions on types of content controls, control events, and types of fields used. New capabilities highlighted - plain text content control API, new fields, and six events in preview. Auto launching add-ins. This survey assessed an understanding of auto launch for add-ins and scenarios where add-ins should be automatically launched and automatically installed. 

Reviewer: ```suggestion
* **Word Extensibility Updates - April 2023.** Results and insights from two surveys. Structured document content usage in Word add-ins. Questions on types of content controls, control events, and types of fields used. New capabilities highlighted: plain text content control API, new fields, and six events in preview. Auto launching add-ins. This survey assessed an understanding of auto launch for add-ins and scenarios where add-ins should be automatically launched and automatically installed. 
```

> * **Top 5 AppSource Validation Errors December 2022 - February 2023.** The presenter focuses on causes and how to avoid top AppSource validation errors affecting Office Add-ins’ submissions and she highlights supporting AppSource policies. In the last 3 months, top validation errors included: App does not work on Office for Mac, functionality doesn’t work as described, Outlook add-in doesn’t work on Safari, privacy policy unavailable or insufficient, and app provides insufficient value.     

Reviewer: ```suggestion
* **Top 5 AppSource Validation Errors December 2022 - February 2023.** The presenter focuses on causes and how to avoid top AppSource validation errors affecting Office Add-ins’ submissions and she highlights supporting AppSource policies. In the last three months, top validation errors included: App does not work on Office for Mac, functionality doesn’t work as described, Outlook add-in doesn’t work on Safari, privacy policy unavailable or insufficient, and app provides insufficient value.     
```

## Call to action

* Join a community panel – product focused add-in discussion groups.
    * [Outlook add-ins panel](https://ux.microsoft.com/Panel/OutlookAddinDeveloper)
    * [Excel add-ins panel](https://ux.microsoft.com/Panel/ExcelAddinDeveloper)
    * [Word add-ins panel](https://ux.microsoft.com/Panel/WordAddinDeveloper)
    * [PowerPoint add-ins panel](https://ux.microsoft.com/Panel/PowerPointAddinDeveloper)
    * [Samples and docs](https://ux.microsoft.com/Panel/OfficeAddinImproveSamplesDocs)
* [Register for the PnP Recognition Program](https://pnp.github.io/recognitionprogram/) and earn contributor badges.

> * Follow channels in Twitter to see call agendas, important updates, and release announcements.

Reviewer: ```suggestion
* Follow channels on Twitter to see call agendas, important updates, and release announcements.
```

    * [@m365pnp](https://twitter.com/m365pnp)
* Register for the [Microsoft 365 Developer Program](https://aka.ms/m365/devprogram) and get a free E5 developer tenant with instant availability and other assets.
* [Next community call](https://aka.ms/officeaddinscommunitycall) on May 10th at 8:00 AM Pacific Time.
* Please complete the [Office add-in developing experience survey](https://forms.office.com/r/wmzCgccbPa).

## Q&A (Question & Answers)

**What is the recommended front end technology to create an Excel add-in? React, Angular, Blazor? Thanks in advance!**

You can use your preferred framework to develop an Excel add-in. To learn more, see [Develop Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/develop/develop-overview). We also invite you to try out the [Use React to build an Excel task pane add-in](https://learn.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-quickstart-react) and [Use Angular to build an Excel task pane add-in](https://learn.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-quickstart-angular) tutorials.

**When will the FileInsertBase64 be released into full production?**

> TWe're estimating the FileInsertBase64 method to be released in production in two or three months.

Reviewer: ```suggestion
We're estimating the FileInsertBase64 method to be released in production in the third quarter of this calendar year (July-Sept 2023).
```
Normally, we don't state release dates but since you're "estimating" and it answers the Q, I think it's fine. To be safe, I changed to Q3 (which is 2-3 months with a buffer!)
